packages/asset-swapper/src/types.ts

```diff
@@ -213,6 +217,8 @@ export interface SwapQuoterOpts extends OrderPrunerOpts {
     contractAddresses?: ContractAddresses;
     samplerGasLimit?: number;
     liquidityProviderRegistryAddress?: string;
+    rfqtTakerApiKeyWhitelist: string[];
```

Comment:
make this optional

Reply:
Addressed in https://github.com/0xProject/0x-monorepo/pull/2541/commits/5602aacbd21494f38d9ac6b01b6248f552768aa1